### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by romanchenko31**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,11 +2,12 @@
   "name": "react-homework-template",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://goitacademy.github.io/react-homework-template/",
+  "homepage": "https://romanchenko31.github.io/reactHomework-1/",
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.3",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "clsx": "^1.2.1",
```
Comment (Oleksii-Romanchenko) [inline package.json]: why do you need that package, I do not see that you use it

File: src/components/FriendList/FriendList.module.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.friendList{
+  width: 250px;
+  margin: 0px auto;
+  background: #e6dfec;
```
Comment (Oleksii-Romanchenko) [inline src/components/FriendList/FriendList.module.css]: move colour to a constants file

File: src/components/FriendListItem/FriendListItem.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import style from '../FriendListItem/FriendListItem.module.css';
+
+
+const FriendListItem = ({ avatar, name, isOnline, id }) => {
+  return (
+      <li key={id} className={style.item}>
+
+ 
+      {isOnline?<span className={style.isOn}></span>:<span className={style.isOff}></span>}
```
Comment (Oleksii-Romanchenko) [inline src/components/FriendListItem/FriendListItem.jsx]: clause
? (if true)
: (if false)

File: package.json (hunk 1 above)
Comment (romanchenko31) [reply package.json]: hz. i wanted use it. but i changed my mind

Comment (romanchenko31) [reply package.json]: i remove it

File: src/components/FriendList/FriendList.module.css (hunk 2 above)
Comment (romanchenko31) [reply src/components/FriendList/FriendList.module.css]: i did it

File: package.json (hunk 1 above)
Comment (Oleksii-Romanchenko) [reply package.json]: Yes remove please and check that everything works

File: src/components/FriendListItem/FriendListItem.jsx (hunk 3 above)
Comment (romanchenko31) [reply src/components/FriendListItem/FriendListItem.jsx]: i did it

**Review by Oleksii-Romanchenko — APPROVED** [review]

**Pull request merged**